Buildings/Experimental/DHC/Networks/Connections/Examples/Connection2PipeExample.mo

```diff
@@ -0,0 +1,184 @@
+within Buildings.Experimental.DHC.Networks.Connections.Examples;
+model Connection2PipeExample
+  "Example model that showcases Connection2PipeAutosize and Connection2PipePlugFlow models"
+  extends Modelica.Icons.Example;
+    package MediumW = Buildings.Media.Water "Medium model";
+  parameter Modelica.Units.SI.Length dhDis(
+    fixed=false,
+    start=0.01,
+    min=0.001) "Hydraulic diameter (assuming a round cross section area)";
+  parameter Modelica.Units.SI.Length dhRet(
+    fixed=false,
+    start=0.01,
+    min=0.001) "Hydraulic diameter (assuming a round cross section area)";
+  parameter Modelica.Units.SI.MassFlowRate mDis_flow_nominal=1
+    "Nominal mass flow rate in the distribution line";
+  parameter Modelica.Units.SI.MassFlowRate mCon_flow_nominal=0.5
+    "Nominal mass flow rate in the connection line";
+  parameter Modelica.Units.SI.Length lDis=100
+    "Length of the distribution pipe before the connection";
+  Buildings.Experimental.DHC.Networks.Connections.Connection2PipeAutosize
+    connection2PipeAutosize(
+    redeclare package Medium = MediumW,
```

Comment:
Corrected the formatting to make it easier to read.

Reply:
I formatted all the code trying to make it more readable